frontend/www/js/omegaup/components/problem/List.vue

```diff
@@ -93,3 +101,13 @@ export default class List extends Vue {
   }
 }
 </script>
+
+<style scoped>
+.form-control {
+  max-width: 14rem;
+}
+.searchbar-width{
+  min-width: 20rem;
+}
+
```

Comment:
Aquí para que tenga efecto el estilo habrá que agregar la clase que habías usado en el otro componente de esta forma:
```suggestion

.searchbar-width .tags-input-wrapper-default {
  min-width: 20rem;
}
```

Sólo habrá que quitarle el `scoped` a la etiqueta de `style`